src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="first-form"
```

Comment:
It's not the best class name. What if we will decide to move this form to another place (this form will not be the first)?